assets/js/blocks/product-collection/inspector-controls/index.tsx

```diff
@@ -3,27 +3,49 @@
  */
 import type { BlockEditProps } from '@wordpress/blocks';
 import { InspectorControls } from '@wordpress/block-editor';
-import { PanelBody } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
+import {
+	// @ts-expect-error Using experimental features
+	// eslint-disable-next-line @wordpress/no-unsafe-wp-apis
+	__experimentalToolsPanel as ToolsPanel,
+} from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
 import { ProductCollectionAttributes } from '../types';
 import ColumnsControl from './columns-control';
 import OrderByControl from './order-by-control';
+import OnSaleControl from './on-sale-control';
+import { setQueryAttribute } from './utils';
+import { DEFAULT_FILTERS, getDefaultSettings } from './constants';
 
 const ProductCollectionInspectorControls = (
 	props: BlockEditProps< ProductCollectionAttributes >
 ) => {
 	return (
 		<InspectorControls>
-			<PanelBody
-				title={ __( 'Settings', 'woo-gutenberg-products-block' ) }
+			<ToolsPanel
+				label={ __( 'Settings', 'woo-gutenberg-products-block' ) }
+				resetAll={ () => {
+					const defaultSettings = getDefaultSettings(
+						props.attributes
+					);
+					props.setAttributes( defaultSettings );
+				} }
 			>
 				<ColumnsControl { ...props } />
 				<OrderByControl { ...props } />
-			</PanelBody>
+			</ToolsPanel>
+
+			<ToolsPanel
+				label={ __( 'Filters', 'woo-gutenberg-products-block' ) }
+				resetAll={ () => {
+					setQueryAttribute( props, DEFAULT_FILTERS );
+				} }
+			>
+				<OnSaleControl { ...props } />
```

Comment:
Do we need to pass all the props down to the OnSaleControl component? It looks like the component is only using the `attributes` prop